libdnf/rpm/package_sack.cpp

```diff
@@ -75,6 +75,49 @@ void PackageSack::Impl::make_provides_ready() {
     provides_ready = true;
 }
 
+libdnf::solv::SolvMap * PackageSack::Impl::compute_considered_map(
+    libdnf::solv::SolvMap & considered, libdnf::sack::QueryFlags flags) const {
```

Comment:
The original idea was to allow re-use of the existing (allocated memory) of the considered map in case of changes in exclusions - eg when changing the set of allowed modules. But it is not used now. I can simplify the code. And we will see in the future.